view/frontend/templates/breadcheckout/price.phtml

```diff
@@ -0,0 +1,150 @@
+<?php /** @var $block Bread\BreadCheckout\Block\Product\View */ ?>
+<?= /* @noEscape */ $block->getBeforeHtml(); ?>
+
+<div class="splitpay-clickable-price" style="padding-bottom: 5px"></div>
+
+<script type="text/javascript">
+    require(['jquery', 'Bread_BreadCheckout/js/product/splitpay'], function($, splitpay) {
+
+        document.splitpayconfigureButton = function (items) {
+            var buttonCss           = '<?= /* @noEscape */ $block->getButtonDesign(); ?>';
+            var financingInfo   = <?= /* @noEscape */ $block->getFinancingJson(); ?>;
+            var breadConfig = {
+                <?php if (!$block->isHealthcare()) : ?>
+                items: items,
+                <?php else : ?>
+                customTotal: items[0]['price'],
+                <?php endif; ?>
+                discounts: <?= /* @noEscape */ $block->getDiscountDataJson(); ?>,
```

Comment:
Should discounts be removed for healthcare merchants? That's how it's set up for WC currently.

Reply:
ok I looked at that function itself and all it does if just return an empty array, so there'll never be a discount here. That's something to look at when we refactor though, as to why and if we need to keep it like that haha